database/migrations/2020_05_04_150000_create_bookings_table.php

```diff
@@ -20,16 +20,16 @@ public function up()
             $table->unsignedInteger('amount')->nullable();
             $table->unsignedInteger('total_taxes')->nullable();
             $table->unsignedInteger('total_fees')->nullable();
-            $table->morphs('variation')->nullable();
-            $table->morphs('experience')->nullable();
-            $table->morphs('order')->nullable();
+//            $table->morphs('variation')->nullable();
```

Comment:
@sl0wik how do we want to write tests for these? I'm not all that familiar with tests for these. Wouldn't these require the tipoff packages these models have been moved into? See my addtional comment below as well.

Reply:
Its still to be confimed by @drewroberts https://github.com/tipoff/bookings/issues/42

We can use those rates inside variations.

Reply:
I confirm using Variations and moving in this direction.

Reply:
I created Issue #44 for Variations